docs/source/manual/upgrade-notes/upgrade-notes-2_1_x.rst

```diff
@@ -30,5 +30,24 @@ Details about these changes in Hibernate Validator can be found at:
 .. _upgrade-notes-dropwizard-2_1_x-health:
 
 Migrating from dropwizard-health
-================================================
-TBD
\ No newline at end of file
+================================
+
+Migrating from the HealthBundle
+-------------------------------
+You'll want to remove the ``HealthBundle`` from your `Application`'s ``initialize()`` method, as you no longer need to add a bundle to get the dropwizard-health functionality.
+
+You'll also want to remove a reference to ``HealthConfiguration`` from your `Configuration` class, as it's now part of the base Dropwizard `Configuration`.
+
+Once you do that, and upgrade `2.1.x` you should be able to just rely on the adding the dropwizard-health behavior by adding config under the top-level config ``health``! 
```

Comment:
below is suggested rewording of section. I think a bulleted list with shorter imperative instructions will be clearer and easier to follow.

---
Starting in Dropwizard 2.1, the functionality from the external [dropwizard/dropwizard-health](https://github.com/dropwizard/dropwizard-health) module has been integrated into the main [dropwizard/dropwizard](https://github.com/dropwizard/dropwizard) project. 

Perform the following steps to migrate your application:
- Remove the  ``HealthBundle`` from your `Application`'s ``initialize()`` method.
- Remove the reference to ``HealthConfiguration`` from your application's `Configuration` class.
- Migrate YAML configuration previously associated with your ``HealthConfiguration`` field to instead be nested under a top-level ``health`` field (at the same level as ``server``, ``logging``, and ``metrics``).
  - Review the new and changed configuration options mentioned below.
  - Ensure that you don't already have a custom configuration class member using the ``health`` name or rename it to something else or nest it under another field in order not to conflict with the new top-level field.
- Remove the dependency on the ``io.dropwizard.modules:dropwizard-health`` library from your application's ``pom.xml`` file (or other build dependency definition file).